sklearn/utils/tests/test_extmath.py

```diff
@@ -661,3 +663,20 @@ def test_stable_cumsum():
     assert_array_equal(stable_cumsum(A, axis=0), np.cumsum(A, axis=0))
     assert_array_equal(stable_cumsum(A, axis=1), np.cumsum(A, axis=1))
     assert_array_equal(stable_cumsum(A, axis=2), np.cumsum(A, axis=2))
+
+
+def test_csr_to_array_fallback():
+    X = np.random.rand(100, 10)
```

Comment:
Please make test run deterministically and independently of one another by default:

```python
X = np.random.RandomState(42).rand(100, 10)
```